test/test_intensify/test_successive_halving.py

```diff
@@ -228,58 +306,26 @@ def target(x):
                         status=StatusType.SUCCESS, instance_id=i+1, seed=0,
                         additional_info=None)
 
-        # config2 & config3 should be capped and config1 should still be the incumbent
-        inc, _ = intensifier.intensify(challengers=[self.config2, self.config3],
-                                       incumbent=self.config1,
-                                       run_history=self.rh,
-                                       aggregate_func=average_cost)
+        intensifier._update_stage()
+
+        # config2 should be capped and config1 should still be the incumbent
+        inc, _ = intensifier.eval_challenger(challenger=self.config2,
+                                             incumbent=self.config1,
+                                             run_history=self.rh,
+                                             aggregate_func=average_cost)
 
         self.assertEqual(inc, self.config1)
-        self.assertEqual(self.stats.ta_runs, 2)
+        self.assertEqual(self.stats.ta_runs, 1)
         self.assertEqual(self.stats.inc_changed, 0)
         self.assertEqual(list(self.rh.data.values())[2][2], StatusType.CAPPED)
-        self.assertEqual(list(self.rh.data.values())[3][2], StatusType.CAPPED)
-
-    @attr('slow')
-    def test_intensify_4(self):
-        """
-            test intensify with multiple instance-seed pairs
-        """
-
-        def target(x: Configuration, seed: int, instance: str):
-            if instance == 0:
-                time.sleep(0.5)
-            if x['b'] == 0:
-                time.sleep(0.6)
-            return (x['a'] + 1) / 1000.
-
-        taf = ExecuteTAFuncDict(ta=target, stats=self.stats, run_obj="runtime")
-        taf.runhistory = self.rh
 
-        intensifier = SuccessiveHalving(
-            tae_runner=taf, stats=self.stats,
-            traj_logger=TrajLogger(output_dir=None, stats=self.stats),
-            rng=np.random.RandomState(12345), deterministic=False,
-            instances=[0, 1], n_seeds=2, cutoff=1, instance_order=None)
-
-        # config1 should still be the incumbent - over all instance seed pairs
-        inc, _ = intensifier.intensify(challengers=[self.config1, self.config2],
-                                       incumbent=None,
-                                       run_history=self.rh,
-                                       aggregate_func=average_cost)
-
-        self.assertEqual(inc, self.config1)
-        self.assertEqual(len(self.rh.get_runs_for_config(self.config1)), 4)
-        self.assertEqual(len(self.rh.get_runs_for_config(self.config2)), 1)
-
-    @attr('slow')
-    def test_intensify_5(self):
+    def test_intensify_3(self):
         """
-            test intensify with shuffling instance order every run
```

Comment:
What happened to this test?

Reply:
Same reason as the other. But now I think it should be included back since it is a special case.